### PR TITLE
How to run pluginsync for large tests

### PR DESCRIPTION
**Pull request by kjlyon**
How to steps for updating  plugins to the new large test framework.

### CODE REVIEW

File: Updating_plugins_for_large_tests.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Fork repo and clone snap-pluginsync
+- Run `$ ./pluginsync.sh`
+  - If this errors make sure ssh keys are set up
+  
+###In container:
+- run `$ msync update -f {plugin_name} —noop`
+- to exit container: `$ exit`
+- cd into `modules/{plugin_name}`
+- Commit changes and push pluginsync branch to your github repo
+  - **If you plan to make a PR with this branch**, make sure you set the following variables in .travis.yml
```
Comment (nanliu) [inline Updating_plugins_for_large_tests.md]: in .sync.yml (.travis.yml is derived form the sync config file).

File: Updating_plugins_for_large_tests.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- git fetch and git checkout the pluginsync branch that you just pushed 
+- Run large tests with `$ make test-large`
+  - To run in debug mode so you can examine the container while still running,
+    run `$ DEMO=true make test-large`
+  - To examine the container, in a separate terminal run 
+    `$ docker exec -it $(docker ps | sed -n 's/\(\)\s*intelsdi\/snap.*/\1/p') /bin/bash`
+  - To view Snap daemon log, in separate terminal run,
+    `$ docker logs $(docker ps | sed -n 's/\(\)\s*intelsdi\/snap.*/\1/p’)`
+
+- If your plugin has a large_tests.sh or something similar in its /scripts directory, delete that. It is nolonger necessary.
```
Comment (nanliu) [inline Updating_plugins_for_large_tests.md]: no longer

File: Updating_plugins_for_large_tests.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#How to run Pluginsync for large tests
+The following steps are necessary for updating plugins to the new large test framework. 
+
+###In the snap-pluginsync repo:
+- Fork and clone repo of plugin you want to run large tests on
+- Fork repo and clone snap-pluginsync
+- Run `$ ./pluginsync.sh`
+  - If this errors make sure ssh keys are set up
```
Comment (nanliu) [inline Updating_plugins_for_large_tests.md]: maybe add this step before running pluginsync:

* make sure ssh-agent is running and github ssh key is load by running `ssh-add -l` (it should list your key).

File: Updating_plugins_for_large_tests.md (hunk 2 above)
Comment (nanliu) [inline Updating_plugins_for_large_tests.md]: After the PR is created, make sure:

* large test is part of travis ci test matrix
* large test should load and verifies at least one task

Comment (kjlyon) [inline Updating_plugins_for_large_tests.md]: @nanliu, why would you do this after creating the PR and not before?

Comment (nanliu) [inline Updating_plugins_for_large_tests.md]: I mean after the PR is created, _check the travis ci output_ to verify the following items are showing up. (the change to the config files should be made prior to submitting the PR)

File: Updating_plugins_for_large_tests.md (hunk 1 above)
Comment (kjlyon) [inline Updating_plugins_for_large_tests.md]: 👍  I updated based on your suggested changes

File: Updating_plugins_for_large_tests.md (hunk 2 above)
Comment (kjlyon) [inline Updating_plugins_for_large_tests.md]: Got it. I added a couple lines to express this. Once I get your LGTM I'll merge.

Comment (nanliu) [inline Updating_plugins_for_large_tests.md]: LGTM

**Pull request merged**